status-im/status-legacy
# [perf] upgrade realm and improve schemas/queries

* upgrade to realm 2.28 to benefit from perf improvements
* remove user-statuses and replace by seen and outgoing-status fields
to get rid of a lot of bloat queries and computations and in-queries which don't
work in this version of realm
* remove unused seen message, bottom-infos

## Tests
Tested on Android

Tested:

* send and receive messages from public and private chats

To test:

* start from nightlies with some chats that have unread messages and unsent messages
* install this version and check that the migrations worked

![MV5BZjkyYThhNWMtYWRiZi00ZjQyLThiYzYtZjkwYmVkYTRlYTIwXkEyXkFqcGdeQXVyNTk1ODg5ODA@ _V1_](https://user-images.githubusercontent.com/1181225/58755963-05bf8b00-84ef-11e9-9e3c-a7543a76af0f.jpg)

status: wip
